### PR TITLE
SUMO-231059: update doc for 10 v2 apps of phase 1

### PR DESCRIPTION
**Pull request by shivani-sumo**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/saas-cloud/acquia.md [hunk 1]
```diff
@@ -251,10 +251,12 @@ To configure Acquia log forwarding, follow the instructions in the Acquia [docum
 
 This section provides instructions on how to install the Acquia App, as well as examples of each of the dashboards. The app's pre-configured searches and dashboards provide easy-to-access visual insights into your data.
 
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing Acquia Dashboards
```
Comment (JV0812) [inline docs/integrations/saas-cloud/acquia.md:256]:
```suggestion
## Viewing Acquia dashboards
```

File: docs/integrations/saas-cloud/airtable.md [hunk 2]
```diff
@@ -74,15 +74,11 @@ _sourceCategory="Labs/airtable"
 
 Before you begin, collect logs from Airtable API and ingest them into Sumo Logic. Refer to the [Airtable API Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/airtable-source/) to create the source and use the same source category while installing the app.
 
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing Airtable Dashboards​
```
Comment (JV0812) [inline docs/integrations/saas-cloud/airtable.md:79]:
```suggestion
## Viewing Airtable dashboards​
```

File: docs/integrations/saas-cloud/airtable.md [hunk 3]
```diff
@@ -74,15 +74,11 @@ _sourceCategory="Labs/airtable"
...
 
-* All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
-
- You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
-
-* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+{@import ../../reuse/apps/view-dashboards.md}
 
 ### Overview Dashboard
```
Comment (JV0812) [inline docs/integrations/saas-cloud/airtable.md:83]:
```suggestion
### Overview
```

File: docs/integrations/saas-cloud/akamai-cloud-monitor.md [hunk 4]
```diff
@@ -185,10 +185,12 @@ parse "\"reqMethod\":\"*\"" as method, "\"status\":\"*\"" as status, "\"fwdHost\
 
 Now that you have configured Akamai logs, install the Sumo Logic App for Akamai Cloud Monitor to take advantage of the pre-configured searches and dashboards to analyze your Akamai data.
 
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing Akamai Cloud Monitor Dashboards
```
Comment (JV0812) [inline docs/integrations/saas-cloud/akamai-cloud-monitor.md:190]:
```suggestion
## Viewing Akamai Cloud Monitor dashboards
```

File: docs/integrations/saas-cloud/akamai-datastream.md [hunk 5]
```diff
@@ -86,10 +86,12 @@ Follow the instructions for setting up [Integration for Akamai DataStream 2](htt
 
 Now that you have configured Akamai DataStream 2 logs, install the Sumo Logic App for Akamai DataStream to take advantage of the pre-configured searches and dashboards.
 
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing Akamai DataStream Dashboards​
```
Comment (JV0812) [inline docs/integrations/saas-cloud/akamai-datastream.md:91]:
```suggestion
## Viewing Akamai DataStream dashboards​
```

File: docs/integrations/saas-cloud/armis.md [hunk 6]
```diff
@@ -193,16 +193,11 @@ This section explains how to collect logs from Armis API and ingest them into Su
 
 ## Installing the Armis App
```
Comment (JV0812) [inline docs/integrations/saas-cloud/armis.md:194]:
```suggestion
## Installing the Armis app
```

Comment (JV0812) [reply docs/integrations/saas-cloud/armis.md:194]: Kindly follow the same for all the docs.

File: docs/integrations/saas-cloud/armis.md [hunk 7]
```diff
@@ -193,16 +193,11 @@ This section explains how to collect logs from Armis API and ingest them into Su
...
 
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing the Armis Dashboards​
```
Comment (JV0812) [inline docs/integrations/saas-cloud/armis.md:198]:
```suggestion
## Viewing the Armis dashboards​
```

File: docs/integrations/saas-cloud/armis.md [hunk 8]
```diff
@@ -193,16 +193,11 @@ This section explains how to collect logs from Armis API and ingest them into Su
...
 
-* All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
-
- You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
-
-* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
-
+{@import ../../reuse/apps/view-dashboards.md}
 
 ### Alerts Overview Dashboard
```
Comment (JV0812) [inline docs/integrations/saas-cloud/armis.md:202]:
```suggestion
### Overview
```

File: docs/integrations/security-threat-detection/akamai-security-events.md [hunk 9]
```diff
@@ -93,13 +93,11 @@ To begin collecting logs for the Sumo App for Akamai Security Events:
 
 This section has instructions for installing the Sumo Logic App for Akamai Security Events.
 
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing Akamai Security Events Dashboards
```
Comment (JV0812) [inline docs/integrations/security-threat-detection/akamai-security-events.md:98]:
```suggestion
## Viewing Akamai Security Events dashboards
```

File: docs/integrations/saas-cloud/armis.md (hunk 8 above)
Comment (JV0812) [reply docs/integrations/saas-cloud/armis.md:202]: Please maintain the same for the other dashboard titles as well.

**Review by JV0812 — APPROVED** [review]
Minor fixes

File: docs/integrations/saas-cloud/armis.md (hunk 8 above)
Comment (shivani-sumo) [reply docs/integrations/saas-cloud/armis.md:202]: Here, we need to keep it as **Alerts Overview Dashboard**, as this is the name for the dashboard. 
This app has two dashboards containing an overview in its title(Alerts Overview, Device Overview).

**Review by kimsauce — COMMENTED** [review]
@shivani-sumo @JV0812 Can you clarify how we should use these going forward? We should mark those in comments in those two files.
- /docs/reuse/apps/app-install-v2.md
- /docs/reuse/apps/app-install.md

**Review by kimsauce — CHANGES_REQUESTED** [review]
@shivani-sumo @JV0812 Can you clarify how we should use these going forward? We should mark those in comments in those two files.

/docs/reuse/apps/app-install-v2.md
/docs/reuse/apps/app-install.md

**Comment by JV0812** [comment]
@kimsauce there is some changes in the Installation steps for V2 apps. Currently, we are only updating the UI for 30 apps so  the new reuse file (v2) created will be added to the new apps. Going forward when the installation process is changed for all the apps then we can delete the existing reuse file.

**Comment by JV0812** [comment]
The intent of this PR is covered in the PR #3432.

**Pull request closed without merging**